src/Razor/src/Microsoft.AspNetCore.Razor.Tools/GenerateCommand.cs

```diff
@@ -170,6 +174,22 @@ protected override bool ValidateArguments()
                 RazorProjectFileSystem.Create(projectDirectory),
             });
 
+            var success = true;
+            var csharpLanguageVersion = LanguageVersion.Default;
+            if (CSharpLanguageVersion.HasValue())
+            {
+                var rawLanguageVersion = CSharpLanguageVersion.Value();
+                if (!LanguageVersionFacts.TryParse(CSharpLanguageVersion.Value(), out var parsedLanguageVersion))
```

Comment:
@jasonmalinowski how sad does this make you? `CSharpLanguageVersion` is the raw `$(LangVersion)` provided from MSBuild.